tasks/devcontainer.py

```diff
@@ -93,10 +96,9 @@ def setup(
         "datadog_agent_devcontainer",
     ]
     devcontainer["features"] = {}
-    devcontainer["remoteUser"] = "datadog"
     devcontainer["mounts"] = [
         "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind,consistency=cached",
-        "source=${localEnv:HOME}/.ssh,target=/home/vscode/.ssh,type=bind,consistency=cached",
+        "source=${localEnv:HOME}${localEnv:USERPROFILE}/.ssh,target=/home/vscode/.ssh,type=bind,consistency=cached",
```

Comment:
This fallback example is also in the documentation https://containers.dev/implementors/json_reference/#variables-in-devcontainerjson